agenda-connectors-webapp/src/main/resources/locale/portlet/AgendaConnectors_en.properties

```diff
@@ -1,10 +1,10 @@
 #####################################################################################
 #                                    Google Connector                               #
 #####################################################################################
-agendaConnectors.googleCalendarr=Google calendar
+agendaConnectors.googleCalendar=Google calendar
```

Comment:
IMHO these labels should remain with prefix agenda.* since they are mentioned from agenda portlet otherwises they will not be detected